spec/values/calculation/acos.hrx

```diff
@@ -0,0 +1,134 @@
+<===> README.md
+Most of the same behavior tested for `calc()` applies to `acos()`, but for
+terseness' sake isn't tested explicitly.
+
+<===>
+================================================================================
+<===> less_than_negative_one/input.scss
+a {b: acos(-2)}
+
+<===> less_than_negative_one/output.css
+a {
+  b: calc(NaN * 1deg);
```

Comment:
@nex3: Is this behavior intended? This matches the behavior of `math.acos(-2)`, so it's correct by the spec, but it seems odd to have a calculation function simplify to a _different_ calculation function.

Reply:
Yes—it actually simplifies to a Number with value `NaN` and numerator unit `deg`, which is then _serialized_ as `calc(NaN * 1deg)`.